src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderTestTrimInOption.java

```diff
@@ -0,0 +1,30 @@
+/*
+ImportOrder
+option = above\t
+groups = java, org
+ordered = (default)true
+separated = true
+separatedStaticGroups = (default)false
+caseSensitive = (default)true
+staticGroups = (default)
+sortStaticImportsAlphabetically = (default)false
+useContainerOrderingForStatic = (default)false
+tokens = (default)STATIC_IMPORT
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
+
+import static java.lang.Math.*; // ok
```

Comment:
No ok comments please in new test inputs.
Remove all of them.
We are ok to keep only one if no violation is present.

Reply:
really sorry, i missed that while copy paste from other file :)

Reply:
Done, removed